jmix-core/core/src/main/java/io/jmix/core/impl/importexport/EntityImportExportImpl.java

```diff
@@ -752,6 +753,9 @@ protected Collection<Object> createNewCollection(MetaProperty metaProperty) {
     @Nullable
     protected Object findReferenceEntity(Object entity, EntityImportPlanProperty importPlanProperty, SaveContext saveContext,
                                          Set<Object> loadedEntities) {
+        if (!isStored(entity)) {
```

Comment:
Added because of `FilterConfigurationModel#rootCondition` field that causes an exception when deserializing in REST API.